frontend/src/pages/admin/Orders.jsx

```diff
@@ -16,14 +16,14 @@ import { Navigate } from 'react-router-dom';
 
 export const Orders = () => {
     const defaultOrder = {
-        clientId: 1,
-        masterId: 1,
-        cityId: 1,
+        client_id: 1,
```

Comment:
I guess if it is some default enitity
Values should be null, not 1, not 0, not ''.
A null means empty, some value that is not set.